lib/miq_automation_engine/engine/drb_remote_invoker.rb

```diff
@@ -60,10 +64,15 @@ def drb_cache_timeout
       1.hour
     end
 
+    def api_token
+      tm = TokenManager.new("api")
+      tm.gen_token("api", :userid => @workspace.ae_user.userid)
```

Comment:
I think thins might be just a typo. Replacing it to what I wrote makes this PR work nicely. :-)